src/dataframe/io.jl

```diff
@@ -946,6 +970,8 @@ function filldf!(df::DataFrame,
         end
     end
 
+    #TODO implement 'usecols'-optoins
```

Comment:
`filldf!` was added by @johnmyleswhite in a commit (https://github.com/JuliaStats/DataFrames.jl/commit/c7b1f315a89118bb1385758b1a0781e131937e32) that didn't use nor export it. John, do you think this code is still useful?